src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -108,6 +108,11 @@ def load_arguments(self, _):
         c.argument('enable_managed_identity', action='store_true')
         c.argument('assign_identity', type=str, validator=validate_assign_identity)
         c.argument('disable_sgxquotehelper', action='store_true')
+        c.argument('appgw_name', options_list=['--appgw-name'])
```

Comment:
What's other parameters except `--appgw-subnet-cidr` for ? There is no introduction for them.

Reply:
these parameters were added before but I didn't make change in this file.